app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -150,25 +217,28 @@ def save_container_routes_inventory(ems, hashes, target = nil)
     store_ids_for_new_records(ems.container_routes, hashes, :ems_ref)
   end
 
-  def save_container_nodes_inventory(ems, hashes, target = nil)
-    return if hashes.nil?
-    target = ems if target.nil?
-
-    ems.container_nodes.reset
-    deletes = if target.kind_of?(ExtManagementSystem)
-                :use_association
-              else
-                []
-              end
-
-    save_inventory_multi(ems.container_nodes, hashes, deletes, [:ems_ref],
-                         [:labels, :tags, :computer_system, :container_conditions,
-                          :additional_attributes], [:namespace])
-    store_ids_for_new_records(ems.container_nodes, hashes, :ems_ref)
+  def graph_container_nodes_inventory(ems, hashes, target = nil)
+    hashes.to_a.each do |h|
+      h = h.except(:labels, :tags, :additional_attributes) # TODO children
+      h = h.except(:namespace)
+      node = @inv_collections[:container_nodes].lazy_find(h[:ems_ref])
```

Comment:
I now realized I don't know the right way to just link several InventoryObjects when I'm building them all.
In several places, including here, I've used `lazy_find` to pass a parent "pointer" into a child-building method.
But that's probably silly, as I then actually build the node 3 lines down?
Is what I did here even supposed to work (a lazy placeholder existing simultaneously with an actual InventoryObject)?

Is what I do in `graph_computer_system_inventory` better, building them in the right order?

OTOH, if I start building objects together and linking by actually giving the other object, do I weaken the ability to do partial refreshes conferred by lazy_find?
I suppose when I know they always arrive together, e.g. here they're part of a single node's json we get from kubernetes, that's OK.